server/superdesk/tests.py

```diff
@@ -146,12 +146,16 @@ def get_prefixed_url(current_app, endpoint):
 
 
 def setup_db_user(context, user):
+    """
+    Setup the user for the DB authentication.
+    :param context: test context
+    :param dict user: user
+    """
     user = user or test_user
     with context.app.test_request_context(context.app.config['URL_PREFIX']):
         original_password = user['password']
 
-        if user.get('user_type') is None:
-            user['user_type'] = 'administrator'
+        user.setdefault('user_type', 'administrator')
```

Comment:
In general I like this change, very Pythonic. However, it might not exactly equivalent to the old code - can it happen that the `user_type` key exists but is set to `None`? If yes, then the user will not be set to administrator anymore as it was in the old version.